plugin/session_buffer.py

```diff
@@ -259,12 +259,13 @@ def on_diagnostics_async(self, raw_diagnostics: List[Diagnostic], version: Optio
             diagnostics_version = version
             diagnostics = []  # type: List[Tuple[Diagnostic, sublime.Region]]
             for diagnostic in raw_diagnostics:
+                region = range_to_region(Range.from_lsp(diagnostic["range"]), view)
```

Comment:
Pyright doesn't like it:
```
Could not access item in TypedDict ​Pyright:
                   "range" is not a required key in "Diagnostic", so access may result in runtime exception
```

Reply:
But it's just moved code so...

Reply:
Yeah. That’s because of total=False in protocol.py